runtime-light/state/instance-state.h

```diff
@@ -119,7 +120,7 @@ struct InstanceState final : vk::not_copyable {
   JobWorkerClientInstanceState job_worker_client_instance_state{};
   JobWorkerServerInstanceState job_worker_server_instance_state{};
 
-  RegexInstanceState regex_instance_state{};
+  RegexInstanceState regex_instance_state;
```

Comment:
Why?

Reply:
Now it's initialized in `InstanceState`'s constructor, so no need to default-initialize it here.